Core/include/Acts/TrackFinding/TrackSelector.hpp

```diff
@@ -382,7 +384,8 @@ bool TrackSelector::isValidTrack(const track_proxy_t& track) const {
          within(track.loc0(), cuts.loc0Min, cuts.loc0Max) &&
          within(track.loc1(), cuts.loc1Min, cuts.loc1Max) &&
          within(track.time(), cuts.timeMin, cuts.timeMax) &&
-         checkMin(track.nMeasurements(), cuts.minMeasurements);
+         checkMin(track.nMeasurements(), cuts.minMeasurements) &&
+         checkMax(track.nHoles(), cuts.maxHoles);
```

Comment:
This is nice. I see `nholes()` is available from the `TrackProxy` and `VectorTrackContainer`.

We could also add  `nOutliers()`, `nSharedHits()`, and `chi2()`. Maybe for a separate PR?

Reply:
Can add more. No preference if all at once or in separate PRs.

Reply:
My wishful suggestion: If it doesn't slow things down too much, can you do them all? If your other work it would delay this PR, please merge now. Either way, thanks!